ruby.rb

```diff
@@ -7,4 +7,26 @@
 # Add comments to your code explaining your approach
 
```

Comment:
I just remembered I told myself I was going to go back and leave comments on the ruby testing part and I completely forgot so I will do that now... honestly I didn't look at the testing file except to get the method names, I was fully relying on the console to tell me what to do

// attr_accessor bc that's the getter/setter for the instance variables. And I think it allows the user to use dot notation when referencing the instance vars outside the class
// only model needs to be passed when instantiating a new instance of Bike bc all other instance vars have default values (and it makes sense for them to)
// current_speed method returns @speed bc that instance var holds the speed val
// pedal_faster method, when called, is intended to be passed a positive num that will be added to whatever val @speed is at while simultaneously resetting the val of @speed
// brake method works in a similar way; brake is intended to be passed a positive num that will be subtracted from the val of @speed. I did have to go back after the test continued to fail and refactor my code to account for vals larger than @speed and I used a ternary statement to do so
// the bike_info method is just regurgitating the current value of all the instance vars in a cutie little string interpolated statement